open-metadata-implementation/access-services/glossary-view/glossary-view-client/src/main/java/org/odpi/openmetadata/accessservices/glossaryview/client/GlossaryViewClient.java

```diff
@@ -0,0 +1,599 @@
+
+    private static final String GET_GLOSSARY = PATH_ROOT + "/glossaries/{2}";
+
+    private static final String GET_GLOSSARIES = PATH_ROOT + "/glossaries" + PARAM_DELIMITER + "from={2}&size={3}";
+    private static final String GET_TERM_HOME_GLOSSARY = PATH_ROOT + "/terms/{2}/home-glossary";
+    private static final String GET_CATEGORY_HOME_GLOSSARY = PATH_ROOT + "/categories/{2}/home-glossary";
+    private static final String GET_GLOSSARY_EXTERNAL_GLOSSARIES = PATH_ROOT + "/glossaries/{2}/external-glossaries" + PARAM_DELIMITER + PAGINATION;
+
+    private static final String GET_CATEGORY = PATH_ROOT + "/categories/{2}";
+    private static final String GET_CATEGORIES_OF_GLOSSARY = PATH_ROOT + "/glossaries/{2}/categories";
+    private static final String GET_SUBCATEGORIES = PATH_ROOT + "/categories/{2}/subcategories";
+    private static final String GET_CATEGORY_EXTERNAL_GLOSSARIES = PATH_ROOT + "/categories/{2}/external-glossaries" + PARAM_DELIMITER + PAGINATION;
+
+    private static final String GET_TERM = PATH_ROOT + "/terms/{2}";
+    private static final String GET_TERMS_OF_GLOSSARY = PATH_ROOT + "/glossaries/{2}/terms";
+    private static final String GET_TERMS_OF_CATEGORY = PATH_ROOT + "/categories/{2}/terms";
+    private static final String GET_TERM_EXTERNAL_GLOSSARIES = PATH_ROOT + "/terms/{2}/external-glossaries" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_RELATED_TERMS = PATH_ROOT + "/terms/{2}/see-also" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_SYNONYMS = PATH_ROOT + "/terms/{2}/synonyms" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_ANTONYMS = PATH_ROOT + "/terms/{2}/antonyms" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_PREFERRED_TERMS = PATH_ROOT + "/terms/{2}/preferred-terms" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_REPLACEMENT_TERMS = PATH_ROOT + "/terms/{2}/replacement-terms" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_TRANSLATIONS = PATH_ROOT + "/terms/{2}/translations-terms" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_IS_A = PATH_ROOT + "/terms/{2}/is-a" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_VALID_VALUES = PATH_ROOT + "/terms/{2}/valid-values" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_USED_IN_CONTEXTS = PATH_ROOT + "/terms/{2}/used-in-contexts" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_ASSIGNED_ELEMENTS = PATH_ROOT + "/terms/{2}/assigned-elements" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_ATTRIBUTES = PATH_ROOT + "/terms/{2}/attributes" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_SUBTYPES = PATH_ROOT + "/terms/{2}/subtypes" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_TYPES = PATH_ROOT + "/terms/{2}/types" + PARAM_DELIMITER + PAGINATION;
+
+    /**
```

Comment:
Add constructor with no userId and password

Reply:
Added